Translation Script/Map117.csv

```diff
@@ -1,63 +1,64 @@
-"回想部屋","回想部屋"
-"回想猫","回想猫"
-"\n<回想猫>やぁ。ここは回想部屋だよ！","\n<回想猫>やぁ。ここは回想部屋だよ！"
-"回想部屋について","回想部屋について"
-"アイテムを売ってほしい","アイテムを売ってほしい"
-"全開放したい","全開放したい"
-"雑談しよう","雑談しよう"
-"何でもない","何でもない"
+"回想部屋","Recollection Room"
+"回想猫","Recollection Cat"
+"\n<回想猫>やぁ。ここは回想部屋だよ！","\n<Recollection Cat>Hi. This is the recollection room!"
```

Comment:
If it was me, I'd put in a cat themed pun in here instead of just "hi". I can't think of anything solid right now, but anything that could make it more cat like ^^

Otherwise, feel free to just directly translate it as "Hello-nya!"